packages/wallet-connectors/src/BrowserWallet.ts

```diff
@@ -45,6 +51,7 @@ export class BrowserWalletConnector implements WalletConnector, WalletConnection
                 .then((a) => delegate.onAccountChanged(this, a))
                 .catch(console.error)
         );
+        this.grpc = new ConcordiumGRPCClient(this.client.grpcTransport);
```

Comment:
This doesn't update when the user switches network/chain. I pushed a commit to this PR to expose the transport object instead of a stored client.

Reply:
Well the transport forwards any request to the wallet, whatever it's configured with, so I don't see the problem in the way it was implemented? Did you experience errors?

https://github.com/Concordium/concordium-browser-wallet/blob/main/packages/browser-wallet-api/src/gRPC-transport.ts for reference

Reply:
Previously the GRPC client that was exposed was also only created when the wallet API was instantiated, so there's no difference from then to now besides the fact that the transport used to be wrapped in a grpc client.

If we go with your suggestion (which I'm also fine with), won't we need to make a similar adjustment to the wallet connect version?

Reply:
There is no counterpart for WalletConnect. As the docs state the method is included only because it's in the BW API.

Reply:
Alright... let's do it then.